shorter.go

```diff
@@ -20,20 +20,26 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-// charset consists of alphanumeric characters with some characters removed due to them being to similar in some fonts.
-const charset = "abcdefghijkmnopqrstuvwxyz23456789ABCDEFGHJKLMNPQRSTUVWXYZ"
-const mutationsLen1 = len(charset)
-const mutationsLen2 = len(charset) * len(charset)
-const mutationsLen3 = len(charset) * len(charset) * len(charset)
+const (
+	// charset consists of alphanumeric characters with some characters removed due to them being to similar in some fonts.
+	charset       = "abcdefghijkmnopqrstuvwxyz23456789ABCDEFGHJKLMNPQRSTUVWXYZ"
+	mutationsLen1 = len(charset)
+	mutationsLen2 = len(charset) * len(charset)
+	mutationsLen3 = len(charset) * len(charset) * len(charset)
+)
+
 const debug = true
 
-// Config contains all valid fealds from a shorter config file
+// dateFormat specifies the format in which date and time is represented.
+const dateFormat = "Mon 2006-01-02 15:04 MST"
```

Comment:
I looked to see if this was a standard format, but seems like you've created a new standard to unify them all :)

From `go doc time.ANSIC`:

```go
const (
	ANSIC       = "Mon Jan _2 15:04:05 2006"
	UnixDate    = "Mon Jan _2 15:04:05 MST 2006"
	RubyDate    = "Mon Jan 02 15:04:05 -0700 2006"
	RFC822      = "02 Jan 06 15:04 MST"
	RFC822Z     = "02 Jan 06 15:04 -0700" // RFC822 with numeric zone
	RFC850      = "Monday, 02-Jan-06 15:04:05 MST"
	RFC1123     = "Mon, 02 Jan 2006 15:04:05 MST"
	RFC1123Z    = "Mon, 02 Jan 2006 15:04:05 -0700" // RFC1123 with numeric zone
	RFC3339     = "2006-01-02T15:04:05Z07:00"
	RFC3339Nano = "2006-01-02T15:04:05.999999999Z07:00"
)
```